provisioners/windows/modules/Bootstrap/Public/Bootstrap-AZPuppet.ps1

```diff
@@ -0,0 +1,131 @@
+﻿function Bootstrap-AzPuppet {
```

Comment:
This is required to run the pester tests in windows 11, since it calls several custom functions inside of this module. For now, I've added the original `bootstrap.psm1` [here](https://github.com/mozilla-platform-ops/ronin_puppet/pull/418/commits/23b73b1d492fdeea061a414cbafe8c0844237513).